examples/gateway-httproute-rewrite-path.yaml

```
@@ -100,3 +100,25 @@ spec:
     - name: echo
       kind: Service
       port: 80
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: rewrite-path-prefix
+spec:
+  parentRefs:
+    - name: kong
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /old-prefix
+      filters:
+        - type: URLRewrite
+          urlRewrite:
+            path:
+              type: ReplacePrefixMatch
+              replacePrefixMatch: "/echo"
```

```suggestion
              replacePrefixMatch: /echo
```
to be consistent with other paths in this yaml that are not quoted. I suppose this quote is meaningless here, isn't it?